src/pcm-memory.cpp

```diff
@@ -81,7 +81,7 @@ bool anyPmem(const ServerUncoreMemoryMetrics & metrics)
 
 bool skipInactiveChannels = true;
 
-void print_help(const string prog_name)
+void print_help(const string & progname)
 {
     cerr << "\n Usage: \n " << progname
```

Comment:
How the heck did this ever compile? Is there a global progname variable or so?

Reply:
the change of the parameter name is not necessary and requires changing the name in other code lines. Could you please keep the old name?